packages/lit-virtualizer/src/LitVirtualizer.ts

```diff
@@ -113,6 +113,10 @@ export class LitVirtualizer extends LitElement {
     render(): TemplateResult {
         const { items, _renderItem, keyFunction } = this;
         const itemsToRender = [];
+        // TODO (graynorton): Is this the best / only place to ensure
+        // that _last isn't outside the current bounds of the items array?
+        // Not sure we should ever arrive here with it out of bounds.
+        // Repro case for original issue: https://tinyurl.com/yes8b2e6
```

Comment:
might be worth conditionally logging in the case where this is needed for debugging live sites.